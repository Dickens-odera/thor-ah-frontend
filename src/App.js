import React from 'react';

import './styles/App.css';
import AppRoutes from './AppRoutes';

const App = () => (
  <div className="App">
    <AppRoutes />
  </div>
)

export default App;
