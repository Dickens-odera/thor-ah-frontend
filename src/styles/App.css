* {
  font-family: 'Lato', sans-serif;
  box-sizing: border-box;
  outline: none;
}
.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 5vmin;
}

.App {
  box-sizing: border-box;
  outline: none;
}

.clear::after {
  content: '';
  display: block;
  clear: both;
}

.left {
  float: left;
}

.right {
  float: right;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}


/* USERS PROFILE PAGE */
.user-profile {
  margin: 50px auto;
  min-height: 100%;
  padding: 20px;
  width: 80%;
}

.faTwitter {
  color: #1DA1F2;
  font-size: 1.4em;
}

.faFacebook {
  color: #3B5998;
  font-size: 1.4em;
}

.faLinkedin {
  color: #0077B5;
  font-size: 1.4em;
}