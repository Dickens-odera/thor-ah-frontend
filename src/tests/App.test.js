import React from "react";
import ReactDOM from "react-dom";
import { create } from 'react-test-renderer';
import App from "../App";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("create a snapshot", () => {
  const c = create(<App />);
  expect(c.toJSON()).toMatchSnapshot();
});

