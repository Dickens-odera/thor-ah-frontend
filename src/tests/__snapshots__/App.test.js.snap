// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create a snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Authors Haven App is live
    </p>
    <a
      className="App-link"
      href="/"
      onClick={[Function]}
    >
       Home 
    </a>
    <a
      className="App-link"
      href="/sample"
      onClick={[Function]}
    >
       sample 
    </a>
    <a
      className="App-link"
      href="/profile/user"
      onClick={[Function]}
    >
       profile 
    </a>
  </header>
  <div
    className="landing-page"
  >
    <h1>
      Welcome to the Thor-AH LandingPage
    </h1>
  </div>
  <footer
    className="App-footer"
  >
    <h1>
      This is the Footer
    </h1>
  </footer>
</div>
`;
